package.json

```diff
@@ -39,7 +39,8 @@
       {
         "command": "onevscode.configuration-settings",
         "title": "configuration-settings",
-        "category": "ONE"
+        "category": "ONE",
+        "icon" : "$(play-circle)"
```

Comment:
![image](https://user-images.githubusercontent.com/17171963/151299860-9ee045d9-00d3-43cb-8868-44ebf98bfb0e.png)

Icon selected from https://code.visualstudio.com/api/references/icons-in-labels#icon-listing
TODO: add ONE product icon